src/components/backend-ai-webui-styles.ts

```diff
@@ -551,12 +563,13 @@ export const BackendAIWebUIStyles = [
       mwc-select#group-select {
         display: none;
       }
-
+      #project {
+        padding-top: 10px;
+      }
     }
 
     @media screen and (max-width: 1030px) {
-      span.email, #group-select-box > div > p
-       {
+      span.email, p#project {
```

Comment:
In my opinion, we should better make project selecting UI visible unless the width is not too narrow.